src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
```

Comment:
I am moderately against `std::pair<RippleAddress, RippleAddress>`. `RippleAddress`es are confusing enough already without having two easily confused instances passed around in the same object. As Josh says, `.publickey` is a lot easier to code for than `.first`.

I am indifferent toward `std::pair<AnyPublicKey, AnySecretKey>`, because the types will make it clear which is which, but I don't know how much code will have to change to accommodate the different types.